.shippable.yml

```diff
@@ -5,7 +5,7 @@ compiler: gcc
 env:
     global:
         - SDK=0.9.2
-        - SANITYCHECK_OPTIONS=" --inline-logs -R"
+        - SANITYCHECK_OPTIONS=" --inline-logs -N"
```

Comment:
Why remove -R? Should this be part of a separate commit to disable asserts?

Reply:
-R is broken right now, it was disable by mistake, see hunk below adding the options again, rather than hacking -N in, I am removing -R for now until the related issues are fixed and issue is open about this.